From 8b56cc896a7d743d61379d1e538de35cef4ceb7f Mon Sep 17 00:00:00 2001
From: Andrei Vasiliu <andrei.vasiliu@intel.com>
Date: Tue, 5 Apr 2016 16:30:56 +0300
Subject: [PATCH] cmake: Solved issue with nodejs installation path

Changed ${LIB_INSTALL_DIR} with lib, because the variable expands to
/usr/lib, making the install path /usr/usr/lib/node_modules, which
is incorrect. Now the install path is /usr/lib/node_modules.

Signed-off-by: Andrei Vasiliu <andrei.vasiliu@intel.com>
---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 74bae09..d884c52 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -203,11 +203,11 @@ if (SWIG_FOUND)
       # If a CMAKE_INSTALL_PREFIX has NOT been provided, set NODE_MODULE_INSTALL_PATH
       # base on the NODE_ROOT_DIR.
       if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-        set (NODE_MODULE_INSTALL_PATH ${NODE_ROOT_DIR}/${LIB_INSTALL_DIR}/node_modules/jsupm_${libname}/)
+        set (NODE_MODULE_INSTALL_PATH ${NODE_ROOT_DIR}/lib/node_modules/jsupm_${libname}/)
       # If a CMAKE_INSTALL_PREFIX has been provided, set NODE_MODULE_INSTALL_PATH
       # relative to the provided install directory.
       else ()
-        set (NODE_MODULE_INSTALL_PATH ${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}/node_modules/jsupm_${libname}/)
+        set (NODE_MODULE_INSTALL_PATH ${CMAKE_INSTALL_PREFIX}/lib/node_modules/jsupm_${libname}/)
       endif ()
       install(FILES ${CMAKE_CURRENT_BINARY_DIR}/package.json
           DESTINATION ${NODE_MODULE_INSTALL_PATH} COMPONENT ${libname})
-- 
1.9.1

